jta-1_2-standalone/src/main/java/org/jboss/narayana/quickstarts/jta/Counter.java

```diff
@@ -14,23 +14,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Hi Ondra, I think this change looks like code formatting? Perhaps put a separate commit that enables checkstyle for this module?

Reply:
I still wonder a bit whether making the code style changes separate to a checkstyle rule is sustainable :) My preference would also be the code style change to be in a separate commit